scripts/completion/dvc.zsh

```diff
@@ -12,30 +12,36 @@
 
 _dvc_commands() {
   local _commands=(
-    "init:Initialize dvc over a directory (should already be a git dir)."
-    "destroy:Destroy dvc. Will remove all project's information, data files and cache."
-    "add:Add files/directories to dvc."
-    "import:Import files from URL."
+    "add:Take data files or directories under DVC control."
+    "cache:Manage cache settings."
     "checkout:Checkout data files from cache."
-    "run:Generate a stage file from a given command and execute the command."
-    "pull:Pull data files from the cloud."
-    "push:Push data files to the cloud."
-    "fetch:Fetch data files from the cloud."
-    "status:Show the project status."
-    "repro:Reproduce DVC file. Default file name - 'Dvcfile'."
+    "commit:Save changed data to cache and update DVC files."
+    # daemon?
+    "config:Get or set config settings."
+    "destroy:Remove DVC files, local DVC config and data cache."
+    # diff?
```

Comment:
diff is released and should be part of the script I believe

Reply:
Added.